src/main/java/uk/gov/hmcts/reform/pbis/model/EmailTemplateMapping.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.pbis.model;
+
+public class EmailTemplateMapping {
+
+    private String service;
+    private String templateId;
+    private String notifyApiKey;
+    private String welcomeLink;
+
+
+    public String getService() {
+        return service;
+    }
```

Comment:
can spring create this object via constructor?

Reply:
It does, by default constructor. I wasn't able to make it use constructor with params.

Reply:
how about just public fields then? (it's mutable in current state anyway)

Reply:
I'd prefer not to make mutable fields public